README.md

```diff
@@ -1,59 +1,51 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Module 1: Leaderboard
 
-> One paragraph statement about the project.
+> i made up Leaderboard with javascript 
 
 ![screenshot](./app_screenshot.png)
```

Comment:
- [ ] I would recommend updating your project with the right screenshot so that any user can understand how your project looks.